org.eclipse.winery.workflowmodeler/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
```

Comment:
yes. thanks.
I did not found this complaining at first. I'll delete the redundant stars.